src/world/climate.c

```diff
@@ -330,13 +340,14 @@ static int climate_play_thunder(int instanceIndex, int soundId, int volume, int
 
 // rct2: 0x00993C94
 // There is actually a sprite at 0x5A9C for snow but only these weather types seem to be fully implemented
-const rct_weather climate_weather_data[6] = {
-	{ .temp_delta = 10, .effect_level = 0, .gloom_level = 0, .rain_level = 0, .sprite_id = 0x5A96 }, // Sunny
```

Comment:
there's no technical limitation, but it seems an unnecessary change and one that breaks ease of digesting this initialisation.